template/static/js/v_summary.js

```diff
@@ -4,7 +4,7 @@ function comparator(fn) {
     const b1 = fn(b);
     if (a1 === b1) {
       return 0;
-    } else if (a1 < b1) {
+    } if (a1 < b1) {
```

Comment:
what's the rationale of this change?

Reply:
oh, this change was made in alignment of the airbnb javascript style guide. https://github.com/airbnb/javascript#blocks--no-else-return